src/frontend/src/lib/stores/tokens-to-display.store.ts

```diff
@@ -0,0 +1,3 @@
+import { writable } from 'svelte/store';
+
+export const currentTokensKeysStore = writable<string[]>([]);
```

Comment:
what's this? do we need it? what's string?

Reply:
yes, we are using it in the `defineTokensToDisplay` function: it "stores" the last sorting order of the token list. And we update it ONLY when we update the tokens to display. 

In case of pointer events, it is used to compare if the previous order is the same as the new one.

I added some comments in `defineTokensToDisplay`. Shall I add here too?

Reply:
I don't quite understand—why does it need to be global? Is it to save the sorting order when navigating back and forth?

Following is few questions that crossed my mind right now, what do you think of those?

1. Should we consider providing this in a separate PR?

2. Is it possible to improve the type definition, perhaps by creating a specific type for `string`? This could make the code more readable.

3. Instead of `currentTokensKeyStore`, maybe it could be renamed to something more generic, like `tokensSortingStore`, and contain an object that includes the `string[]`. This way, it becomes more descriptive, and if additional sort options are added in the future, the structure would already accommodate them. I generally don't like adding things in advance but maybe here it's interesting?

4. Similarly to the "hide zero balance" filter, should this list of keys be saved in the browsers' cache or those have to be recaculated each time the app boots?

5. Speaking of, should we even think at having the "filter zero balance: boolean" in the same store? Actually not sure it makes sense to have a single object that contains a filter.

Again, just few things that crossed my mind.

Reply:
> Is it to save the sorting order when navigating back and forth?

it is to save the sorting order between re-sorting: this store "stores" the last state of the order of the token list displayed

1. I can separate in another, but I put this one in draft, since it should come before

2. yes, will do, something like `TokenIndexKey` or simply `TokenKey`?

3. agreed!

4. for this one, I would do it in a separate PR, after having discussed it better

5. make sense! will do it in a separate PR

Reply:
Cool!

4. Note again that I'm really unsure if it makes sense or not to save those in idb, just something that crossed my mind.

5. We have to pay super attention to not create an endless loop of hell when if we do this.